config/src/config/execution_config.rs

```diff
@@ -67,6 +69,7 @@ impl Default for ExecutionConfig {
             num_proof_reading_threads: 32,
             paranoid_type_verification: true,
             paranoid_hot_potato_verification: true,
+            discard_failed_blocks: false,
```

Comment:
Is it ok to have as false by default? The cell in AptosVM has default of true, isn't this inconsistent?

Reply:
Ooops, default there to true is wrong, I wanted false. This is SEV mitigation flag